syntax = "proto2";

import "dccl/option_extensions.proto";

package jaiabot.sensor.protobuf;

message AtlasScientificOEMpH
{
    // notional - add units and additional fields as needed
    optional double ph = 1;
    optional double temperature = 2;
}


